packages/epics/src/membership/components/members/member-spaces.tsx

```diff
@@ -52,6 +52,7 @@ export const MemberSpaces = ({ spaces, isLoading }: MemberSpacesProps) => {
         <div className="flex flex-row gap-3 overflow-x-auto">
           {spaces?.map((space) => (
             <Image
+              key={`key ${space.title}`}
```

Comment:
Let's handle these cases like this. By doing so, we can easier find those places when we have proper data fields.

```suggestion
          {spaces?.map((space, index) => (
            <Image
              // TODO: space doesn't have id field yet
              key={index}
```